contrib/p4/docker_files/ptf_tester/tests/test_no_fail.py

```diff
@@ -0,0 +1,172 @@
+# SPDX-FileCopyrightText: 2020-2021 CASTOR Software Research Centre
+# <https://www.castor.kth.se/>
+# SPDX-FileCopyrightText: 2020-2021 Johan Paulsson
+
+# SPDX-License-Identifier: Apache-2.0
+
+import ptf
+from ptf.base_tests import BaseTest
+from ptf import config
+import ptf.testutils as testutils
+from scapy.all import *
+import re
+
+TYPE_MYTUNNEL = 0x1212
+TYPE_IPV4 = 0x0800
+
+
+class MyTunnel(Packet):
+    name = "MyTunnel"
+    fields_desc = [ShortField("pid", 0), ShortField("dst_id", 0)]
+
+    def mysummary(self):
+        return self.sprintf("pid=%pid%, dst_id=%dst_id%")
+
+
+class DataplaneBaseTest(BaseTest):
+    def __init__(self):
```

Comment:
There should be no need to overwrite `__init__` in this class and similar cases.